tutorials/roofit/rf611_weightedfits.C

```diff
@@ -108,7 +108,11 @@ int rf611_weightedfits(int acceptancemodel=2) {
 
   //number of pseudoexperiments (toys) and number of events per pseudoexperiment
   constexpr unsigned int ntoys = 500;
+#if defined(_WIN32) && !defined(__x86_64__)
```

Comment:
I don't understand why changing the number of events for the dataset used for fitting each toy fixes the problem!

Reply:
> I don't understand why changing the number of events for the dataset used for fitting each toy fixes the problem!

`data.add(RooArgSet(costheta, weight), weight.getVal());` makes the memory increase at each iteration and reach the limit for 32 bit

Reply:
IIRC, on Win32, the user-kernel split for the virtual address space is at 2GB, which means 2GB usable for the process.  This _might_ work, however, in Linux 32-bit as the user-kernel split is a bit higher (at 3GB).

Is it normal that the process is consuming up to 2GB of memory?  Might be that a memory leak somewhere, @lmoneta?